pg_search/src/api/operator/searchqueryinput.rs

```diff
@@ -105,10 +176,10 @@ pub fn query_input_restrict(
             let (heaprelid, _, _) = find_var_relation(var, info);
             let indexrel = locate_bm25_index(heaprelid)?;
 
-            let query = SearchQueryInput::from_datum((*const_).constvalue, (*const_).constisnull)?;
-            let search_config = SearchConfig::from((query, &indexrel));
+            let search_query_input =
+                SearchQueryInput::from_datum((*const_).constvalue, (*const_).constisnull)?;
```

Comment:
This query will need to be wrapped with `SearchQueryInput::WithIndex {oid, query}` too..  Like the more complex situation for `@@@(anyelement, text)`, this simpler form still needs to know the index oid incase of a sequential scan.